.github/stale.yml

```diff
@@ -0,0 +1,9 @@
+daysUntilStale: 14
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+  - in backlog
```

Comment:
```suggestion
exemptLabels:
  - pinned
  - security
  - in backlog
  - bug
```

Reply:
At first I thought good point, but then I reconsidered, because if somebody opens an improper bug ticket without enough details it will never get closed. Once we start working on it we should move it to the backlog and then it should be protected.

Reply:
I feel like it's generally too early for automation and I'd be more protective on the customer side. So far we try to "protect" us and UX can be broken for customers. With false tags, we can remove them as I understand. It's any way on us to manage the state of the items.